### PR TITLE
Add a couple of movie scrapers.

### PR DESCRIPTION
**Pull request by woodgen**
Adds movie by URL scrapers for AdultEmpire, Iafd and IafdStreaming.

### CODE REVIEW

File: scrapers/AdultDvdMarketPlace.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+xPathScrapers:
+  movieScraper:
+    movie:
+      Name: //meta[@property="og:title"]/@content
+      Director: //span[contains(text(),"Director")]/following-sibling::a/text()
+      Date:
+        selector: //span[contains(text(),"Released")]/following-sibling::text()
+        postProcess:
+          - parseDate: 01/2006
+      Synopsis: //h3[contains(text(), "Description")]/following-sibling::p/text()
```
Comment (bnkai) [inline scrapers/AdultDvdMarketPlace.yml:18]: `/text()` should be removed as it ommits bold or other formatted parts of the paragraph
eg `https://www.adultdvdmarketplace.com/dvd_view_362671.html`
```yaml
      Synopsis: //h3[contains(text(), "Description")]/following-sibling::p
```
is enough

File: scrapers/AdultDvdMarketPlace.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      Studio:
+        Name: //span[@itemprop="brand"]/text()
+      FrontImage: //strong[contains(text(),"Large Front")]/parent::a/@href
+      BackImage: //strong[contains(text(),"Large Back")]/parent::a/@href
+
```
Comment (bnkai) [inline scrapers/AdultDvdMarketPlace.yml:23]: #Last updated date should be added as a comment

File: scrapers/AdultEmpire.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      Date:
+        selector: //small[contains(text(), "Released")]/following-sibling::text()
+        postProcess:
+          - parseDate: Jan 02 2006
+      Synopsis: //h4[contains(@class,"synopsis")]/p/text()
+      Studio:
+        Name: //a[@label="Studio"]/text()
+      FrontImage: //a[@id="front-cover"]/@data-href
+      BackImage: //a[@id="back-cover"]/@href
+
```
Comment (bnkai) [inline scrapers/AdultEmpire.yml:31]: #Last updated date should be added as a comment

File: scrapers/Iafd.yml [hunk 4]
```diff
@@ -91,5 +96,25 @@ xPathScrapers:
...
+           with:
+      Director: $biohead[contains(text(), "Directors")]/$biodata_link
+      Duration: $biohead[contains(text(), "Minutes")]/$biodata
+      Synopsis: //div[@id="synopsis"]/div[@class="padded-panel"]//text()
+      Date:
+        selector: $biohead[contains(text(), "Release Date")]/$biodata[1]
+        parseDate: Jan 2, 2006
+      Studio:
+        Name: $biohead[contains(text(), "Studio")]/$biodata_link|$biohead[contains(text(), "Distributor")]/$biodata_link
+
```
Comment (bnkai) [inline scrapers/Iafd.yml:119]: from some movie we can also  get the aliases eg
`http://www.iafd.com/title.rme/title=conquest/year=1994/conquest.htm`
`http://www.iafd.com/title.rme/title=black+man+made+me+squirt/year=2017/black-man-made-me-squirt.htm`
```yaml
      Aliases:
        selector: //div[@class="col-sm-12"]/dl/dd
        concat: ", "

```

File: scrapers/Iafd.yml [hunk 5]
```diff
@@ -91,5 +96,25 @@ xPathScrapers:
...
 # Last Updated July 2, 2020
```
Comment (bnkai) [inline scrapers/Iafd.yml:120]: date should be update

File: scrapers/AdultDvdMarketPlace.yml (hunk 2 above)
Comment (woodgen) [reply scrapers/AdultDvdMarketPlace.yml:23]: git log `scrapers/AdultDvdMarketPlace.yml`?

**Comment by woodgen** [comment]
Isn't git more than enough and way more reliable than manually editing the change date in the file?
`git log scrapers/foo.yml` tells you also what was changed in the file.

**Comment by bnkai** [comment]
Git is more than good enough for anyone that knows how to use it. 
For troubleshooting with plain users though its a lot easier to help if you know that the user has the most recent version or not. Most plain users download the scrapers they want directly without using git.

**Pull request merged**